### ISSUE TITLE
Implement /tickets/update_many.json

### ISSUE THREAD

**Issue by wakingrufus** [cloudbees/zendesk-java-client#261]
In order to update multiple tickets in one call, Zeondeck provides the `/api/v2/tickets/update_many.json` endpoint. This ticket is to provide java API access to this function

**Comment by wakingrufus** [comment]
PR here: https://github.com/cloudbees/zendesk-java-client/pull/260

**Comment by wakingrufus** [comment]
It seems that the API response contains invalid ticket entities:
```Cannot deserialize value of type `org.zendesk.client.v2.model.Status` from String "Updated": value not one of declared Enum instance names: [new, solved, pending, deleted, open, closed, hold]
```
I will have to comment out some of my test until this is fixed.
Upstream ticket: https://github.com/cloudbees/zendesk-java-client/issues/98

**Comment by aheritier** [comment]
released in 0.8.0
